README.md

```diff
@@ -10,104 +10,150 @@
 
 A GitHub plugin for Mattermost. Supports GitHub SaaS and Enterprise versions.
 
+## Table of Contents
+
+ - [1. Audience](#1-audience)
+ - [2. License](#2-license)
+ - [3. About the GitHub Plugin](#3-about-the-github-plugin)
+ - [4. Before You Start](#4-before-you-start)
+ - [5. Configuration](#6-configuration)
+ - [6. Using the Plugin](#7-using-the-plugin)
+ - [7. Onboarding Your Users](#8-onboarding-your-users)
+ - [8. Slash Commands](#9-slash-commands)
+ - [9. Configuration Notes in High Availability Mode](#4-configuration-notes-in-high-availability-mode)
+ - [10. Frequently Asked Questions](#6-frequently-asked-questions)
+ - [11. Development](#5-development)
+
 ![GitHub plugin screenshot](https://user-images.githubusercontent.com/13119842/54380268-6adab180-4661-11e9-8470-a9c615c00041.png)
 
-## Features
+## Audience
+
+This guide is intended for Mattermost System Admins setting up the GitHub plugin and Mattermost users who want information about the plugin functionality. For more information about contributing to this plugin, visit the [Development section](https://github.com/mattermost/mattermost-plugin-github#development).
+
+## License
+
+This repository is licensed under the Apache 2.0 License.
```

Comment:
Nit pick: Maybe link to `LICENSE`?

Reply:
Added a link to the Apache 2.0 license wording.